server/handlers/default.go

```diff
@@ -20,6 +25,14 @@ import (
 	"github.com/docker/notary/tuf/validation"
 )
 
+const (
+	// ConsistentCacheMaxAge is the Cache-Control header's max age for consistent downloads
+	ConsistentCacheMaxAge int = 30 * 24 * 60 * 60 // 30 days
+
+	// NonConsistentCacheMaxAge is the Cache-Control header's max age for current (non-consistent) downloads
+	NonConsistentCacheMaxAge int = 5 * 60 // five minutes
```

Comment:
Ok, working on the config now, but I think I might skip the `time.Duration`.  Cache control max age is in seconds, so you can't actually pass a smaller unit than second in the configuration (and it makes sense to have the configuration value just be the max-age value, since if users need to configure it they probably are already familiar with and have a value in mind for the max age value).

And converting from seconds to duration and then back, and then rounding and converting to an int seems maybe unnecessary if I only allow ints?